.github/ISSUE_TEMPLATE/malware-check.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Malware Check
+about: Contribute an new automated malware check to PyPI
+
+---
+
+**Describe the proposed check**
+<!-- A clear and concise description of the check to be implemented. Go to https://warehouse.pypa.io/development/malware-checks/ for more information about the malware check types and architecture.-->
+
+**What kind of attack does this check detect?**
+<!-- What threat vector does this check address? -->
+
+**Additional context**
+<!-- Add any other context, links, third-party information, rate limits, etc. about the check here. -->
```

Comment:
May be worth adding a preformed check list, most notably to remind submitters (and reviewers) that the check should be added to the documentation.